_meetups/2023-07-13-qubika.md

```diff
@@ -1,9 +1,13 @@
 ---
 collection: meetups
-title: Qubika
 layout: default
+host: Qubika
+talks:
+ - title: My take on the new Rails 7 defaults
+   speaker: Juan Diego Rosales
+   recording: https://drive.google.com/file/d/1Yod9F1bxIAUpBp8enOq74t_9cFCc2Y7X/preview
+ - title: Exploring the life-changing Magic of Ruby on Rails
+   speaker: Franco Bottero
 ---
 
-{{ page.title }}
-
-<iframe src="https://drive.google.com/file/d/1J0TeBOvg01XC3qrpPJhqAhl8n3Vv5YGc/preview" width="640" height="480" allow="autoplay"></iframe>
+{% include_relative base.html %}
```

Comment:
should we pass host and talks as params?

Reply:
Maybe this should be meetup layout rather than an include?
And on each of the meetup post we could write some text version of the meeting.

Reply:
makes sense